test/Game-test.js

```diff
@@ -23,10 +24,45 @@ describe('it should create a game ', function() {
   })
 
   it('it should start a game', function() {
-    
-    game.start()
+    game.start();
     expect(domUpdates.getPlayerNames).to.have.been.called(1);
+    expect(domUpdates.hideStartScreen).to.have.been.called(1);
+    expect(game.currentWheel).to.have.property('elements').with.lengthOf(6);
+    expect(game).to.have.property('players').with.lengthOf(3);
+    expect(game).to.have.property('puzzles').with.lengthOf(4);
+    expect(game.currentPuzzle).to.be.an.instanceOf(Puzzle);
+    expect(domUpdates.displayPuzzle).to.have.been.called(1);
+  });
+
+  it('it should instanciate three players from Player class', function() {
+    let playerNames = ['Hank', 'Dustin', 'Katiebelle'];
+    game.createPlayers(playerNames);
+    expect(game.players).to.have.lengthOf(3);
+    expect(game.players[0, 1, 2]).to.be.an.instanceOf(Player);
+    expect(game.players[0].name).to.equal('Hank');
   });
 
+  it('should return an array of puzzles', function() {
+    let bankOfPuzzles = game.createPuzzleBank();
+    expect(bankOfPuzzles).to.be.instanceOf(Array);
+    expect(bankOfPuzzles).to.have.lengthOf(96);
+    expect(bankOfPuzzles[0]).to.be.instanceOf(Object);
+    expect(bankOfPuzzles[0]).to.have.property('total_number_of_letters');
+  });
+
+  it('should be able to generate a number given a max range', function() {
+    let maxRange = 45;
+    let num = game.createRandomNumber(maxRange);
+    expect(num).to.be.within(0, maxRange);
+  });
 
+  it('should be able to return an array of 5 puzzles', function() {
+    game.createPuzzles();
+    expect(game.puzzles).to.have.lengthOf(5);
```

Comment:
This is a good start. It would be good to test the content of this array - more than just the length to check if it is the array you expect.